src/api/java/com/minecolonies/api/colony/requestsystem/RequestState.java

```diff
@@ -0,0 +1,98 @@
+package com.minecolonies.api.colony.requestsystem;
+
+import net.minecraft.nbt.NBTTagInt;
+
+import java.util.ArrayList;
+
+/**
+ * Enum used to describe the state of a Request.
+ */
+public enum RequestState
```

Comment:
This class is not supposed to be in this PR yet.
Just the Factories part and the IToken part (and there respective implementations)